src/pages/css/layout/flexbox/index.md

```diff
@@ -8,8 +8,17 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
```

Comment:
Thank you for your Pull Request, Please replace the stub article with your article